contracts/Bank.sol

```diff
@@ -268,7 +244,6 @@ contract Bank is BankStorage, AccessControlEnumerable, Initializable {
             reserve.collateralizationRatio) * 100;
         maxBorrow *= debt.priceGranularity;
         maxBorrow /= collateral.priceGranularity;
-        maxBorrow -= vaults[msg.sender].debtAmount;
```

Comment:
This was the source of the error, subtracting the debt amount was not needed. This was missed since there was no test for the scenario that causes this to be a problem